mongo/operations.go

```diff
@@ -26,6 +32,7 @@ type Operation interface {
 	Error() error
 	Unacknowledged() bool
 	CommandAndCollection() (Command, string)
+	TransactionDetails() (*TransactionDetails, bool)
```

Comment:
any reason to have the ok `bool` here when it's exactly equal to whether `*TransactionDetails` is `nil` or not?

Reply:
Good call. 
Redundant bool removed